### PR TITLE
Add Stories Article (fixes #12379)

### LINKED ISSUE #12379: PBJ: New Article for December
## Description

Create new featured article for go-live **Dec. 5, 2022**

copy (Mozilla only): https://docs.google.com/document/d/1FZibryjAwaAqxAcpRTjRW422QHYmSaCDgLNxEmaUuCU/edit
no images aside from author's headshot

Related request:
- change "Article Credits" to "Article Contributors" for all articles

TBD
- hero image
- background color

### PR DESCRIPTION
**Pull request by maureenlholland**
## One-line summary

Add new featured stories article for December. **Go-live: Monday, Dec. 5**

New bg color and hero image approved by stakeholders. Asking for high-res contributor image, but may not receive one.

## Significant changes and points to review

Changes "Credits" to "Contributors"
Removes image macro from Featured partial, credits, and stories_image macro

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/12379

## Testing

Demo server URL: (or None)

To test this work:

- [x] http://localhost:8000/en-US/stories/raising-technology-eq/
- [x] http://localhost:8000/en-US/stories/ (new landing page featured article, prev. featured article in related section)

No regression after image macro removal:
http://localhost:8000/en-US/stories/joy-of-color/
http://localhost:8000/en-US/stories/art-of-engagement/
http://localhost:8000/en-US/stories/build-together/
http://localhost:8000/en-US/stories/community-champion/

ℹ️ Will be squashed and merged, but leaving separate commits for now to isolate refactoring work
⚠️ DO NOT MERGE until go-live date

### CODE REVIEW

**Comment by maureenlholland** [comment]
TODO:

- [ ] style fixes (landing: related image aspect ratio, article: body overflow, shorter title max width?)
- [ ] confirm background color & match on landing/article pages
- [ ] get true high res contributor image or change image retrieval logic
- [ ] get hero image and caption

**Comment by maureenlholland** [comment]
WIP again: Forgot to update contributor image logic

File: media/css/stories/stories-shared.scss [hunk 1]
```diff
@@ -10,6 +10,51 @@ $image-path: '/media/protocol/img';
...
+    --theme-color-text: #{$color-black};
+}
+
+[class*='theme'] {
+    background-color: var(--theme-color-bg);
+    color: var(--theme-color-text);
+
+    .c-stories-article-blockquote.color-accent {
+        background-color: var(--theme-color-bg);
+
```
Comment (nathan-barrett) [inline media/css/stories/stories-shared.scss:35]: I'm terrible at remembering to use these types of selectors, good job!

**Review by nathan-barrett — APPROVED** [review]
Looks good! 🥌 Only comment would be to reduce the image sizes a bit, but I know these are pretty high quality so it might be hard to trim them down.

**Comment by maureenlholland** [comment]
good call-out re: images, I ran them through the tinypng optimizer, but will try a png format too see if there's improvement

**Comment by alexgibson** [comment]
TinyPNG is also a great option for some lossless compression: https://tinypng.com/ (works with jpgs too)

**Comment by alexgibson** [comment]
Oh sorry, I misread and thought you first said ImageOptim!

**Pull request merged**